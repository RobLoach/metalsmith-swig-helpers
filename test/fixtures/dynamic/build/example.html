<html>
<body>

Encrypted: Ncjji&amp;Qitjb&#39;

&lt;p&gt;This is an example.&lt;/p&gt;


</body>
</html>
