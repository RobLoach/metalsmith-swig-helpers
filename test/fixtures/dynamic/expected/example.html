<html>
<body>

Encrypted: Ncjji&amp;Qitjb&#39;


This is an example.


</body>
</html>
